xbmc/pvr/channels/PVRChannelGroup.h

```diff
@@ -242,6 +248,11 @@ namespace PVR
      */
     //@{
 
+    void Sort();
+    /*!
+     * @brief Sort the group.
+     */
```

Comment:
Doxy before the method declaration, please.

Reply:
Silly, sorry

Reply:
:-)